crates/core/src/wasi_2023_10_18.rs

```diff
@@ -100,66 +100,66 @@ wasmtime::component::bindgen!({
     },
 });
 
-use wasi::cli::terminal_input::TerminalInput;
-use wasi::cli::terminal_output::TerminalOutput;
-use wasi::clocks::monotonic_clock::Instant;
-use wasi::clocks::wall_clock::Datetime;
-use wasi::filesystem::types::{
+use wasi::cli0_2_0_rc_2023_10_18::terminal_input::TerminalInput;
```

Comment:
While that won't make the diff less noisy, would it perhaps make the resulting code a bit nicer if we changed the `use` statements to alias away the version? I.e. start the implementation files with a set statement along the lines of
```rust
use wasi::{
    cli0_2_0_rc_2023_10_18 as wasi_cli,
    clocks0_2_0_rc_2023_10_18 as wasi_clocks,
    ...
};
```

Reply:
Another advantage of this would be that if the bindings generation changes in the future to omit version strings in more scenarios (as I think it could here, because only one version of each interface is actually included in the list of interfaces) we'd only have to change this one place.